k8s/docs/values-stage.yaml

```diff
@@ -1,13 +1,9 @@
-# Default values for airqo-docs.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
 replicaCount: 1
 
 image:
   repository: gcr.io/airqo-250220/airqo-stage-docs
   pullPolicy: Always
-  tag: latest
+  tag: stage-090ad16f-1667922055
```

Comment:
Sets the latest built image

Reply:
Hi @123MwanjeMike, why this change? The default tag for the latest image is always `latest`

Reply:
Yes @NoahNsimbe 
The default image tag (for the k8s manifests) is latest however for ArgoCD to be triggered, it needs to detect a change in the Helm values files thats why we have a step to auto update the image tag in the respective values files.

Reply:
Hi @123MwanjeMike , as the part of "auto update", I am failing to appreciate the "auto" part of this change?